app/views/purchases/_form.html.erb

```diff
@@ -51,18 +51,28 @@
 
 <label class="label">Purchaser</label>
 <div class="field is-grouped">
+
+  <%= form.collection_radio_buttons( :purchaser_id, User.all, :id, :name) do |b| %>
   <div class="control">
-    <%= radio_button_tag(:user_id, User.find_by(name: 'Ellen').id) %>
-    <%= label_tag(:user_ellen, "Ellen") %>
-  </div>
-  <div class="control">
-    <%= radio_button_tag(:user_id, (User.find_by(name: 'Ben')).id) %>
-    <%= label_tag(:user_ben, "Ben") %>
+    <%= b.radio_button %>
+    <%= b.label %>
   </div>
+<% end %>
+</div>
+
+<label class="label">Purchasee</label>
+<div class="field is-grouped">
+
+<%= form.collection_radio_buttons( :purchasee_id, User.all, :id, :name) do |b| %>
+<div class="control">
+  <%= b.radio_button %>
+  <%= b.label %>
+</div>
```

Comment:
See note above about `accepts_nested_attributes_for` - if you use that, there are implications for the view.